Godeps/Godeps.json

```diff
@@ -399,14 +399,14 @@
 			"Rev": "708a7f9f3283aa2d4f6132d287d78683babe55c8"
 		},
 		{
-			"ImportPath": "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime",
-			"Comment": "v1.7.0-alpha.1-493-g32e927f",
-			"Rev": "32e927f4d86cfe7d3a7ad3c231fc445fb01463f5"
+			"ImportPath": "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1",
+			"Comment": "v1.7.0-alpha.3-971-g286bcc6",
+			"Rev": "286bcc6f5c0e4b1bd9315cb5b898966b48efb4b1"
```

Comment:
It seems that `cri-tools` is currently using [1369a263f576270c85974e69b6ad8e7a80a6f494](https://github.com/kubernetes-incubator/cri-tools/blob/master/Godeps/Godeps.json#L729)